script.js

```diff
@@ -0,0 +1,11 @@
+let transform = document.querySelector(".side--me--transform");
+let nickname = document.querySelector(".side--me--nickname");
+
+transform.addEventListener("click",() => {
+    let nicknameInput = prompt("Input Instagram ID")
+    if (nickname == "") {
```

Comment:
자바스크립트라는 언어의 허점을 역이용하면 오히려 더욱 질좋은 코드를 짤 수 있답니다!
```suggestion
if(!nickname) {
```